submissions/ses3332002/friends-app/app.js

```diff
@@ -31,11 +37,14 @@ function render() {
   const contentToRender = document.createDocumentFragment();
   dataToRender.forEach((item) => {
     let card = document.createElement("div");
-    let lastNumberOfAge = item.dob.age.toString().slice(-1);
+    function sexString(age) {
+      const lastNumberOfAge = age.toString().slice(-1);
+      return (lastNumberOfAge == "1") ? "год" : (lastNumberOfAge == "2")||(lastNumberOfAge == "3")||(lastNumberOfAge == "4") ? "года" : "лет";
```

Comment:
Please, use constants for a 'magic numbers'

Reply:
I remade it.